intro-to-ruby.rb

```diff
@@ -1,14 +1,20 @@
 #create a string and store it in a variable
+food = "chicken,french fries,pizza"
+puts " like to eat #{food} after school."
 
 #print out the value of the variable you just created
 
+
 #print the sum of 185 and 14
+puts (185+14)
 
 #print the product of 1.8 and 3.6
-
+puts (1.8*3.6)
 #store the number 78 as a variable
+x="78"
```

Comment:
Here, you're storing 78 as a string, not as a number. Remember that Ruby treats strings and numbers differently. Check out this repl.it and see what happens when you run it. https://repl.it/B0uY/1